### PR TITLE
Feat/eng 995/volunteer paid flow

### PR DESCRIPTION
**Pull request by satanmourner**
**Don't merge this PR**
- [x] reset form after creating a job
- [x] refactor alert modal with using new modal
- [x] updating after sending an offer (FIXME)
- [x] test the complete mission page and confirm the mission page
- [x] test input modal mobile version
- [x] After emerging hot-fixes branch, use `webModal` in `CreateModal` and `ReviewModal`
- [x] final check

### CODE REVIEW

File: src/components/atoms/dropdown-v2/dropdown.types.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { FormGroup } from '../../../core/form/useForm/useForm.types';
 
 export type DropdownItem = {
-  id: string | number;
+  id?: string | number;
```
Comment (s-abbasi) [inline src/components/atoms/dropdown-v2/dropdown.types.ts:4]: The fact that you made id optional is okey, just make sure the values you are passing to dropdonw are unique as they are used for the `key` prop.

We used to have the issue where values received from API were not unique.

File: src/components/organisms/alert-modal/index.tsx [hunk 2]
```diff
@@ -1,43 +1,50 @@
...
   buttons = [],
+  contentClassName = '',
 }) => {
   return (
-    <CardSlideUp open={open} onClose={onClose}>
+    <Modal open={open} onClose={onClose} className={css.modal}>
       <>
-        <div className={css.header}>{header}</div>
+        <div className={`${header ? css.header : css.no__header}`}>
+          <span></span>
```
Comment (s-abbasi) [inline src/components/organisms/alert-modal/index.tsx:23]: Does this empty `<span>` do anything?

File: src/components/atoms/dropdown-v2/dropdown.types.ts (hunk 1 above)
Comment (satanmourner) [reply src/components/atoms/dropdown-v2/dropdown.types.ts:4]: in some sections, you didn't pass ids (static data) and the typescript gave a warning. 
we have 2 options: 1) make it optional and remember the unique id for API data and 2) change all static data and constants and add `id` properties to their data.
whatever you decide.

File: src/components/organisms/alert-modal/index.tsx (hunk 2 above)
Comment (satanmourner) [reply src/components/organisms/alert-modal/index.tsx:23]: a way to handle the grid to 3 sections (left, center. right) with `flex` and maybe later add something on the left side.

**Pull request merged**